### PR TITLE
Tab ui

### PR DESCRIPTION
**Pull request by Nibba2018**
This PR adds support for a TabUI component to Fury's UI module.
The wireframe for the design is as follows:
image

Kindly merge #248 before this.

Thank you.

### CODE REVIEW

**Comment by skoudoro** [comment]
Hi @Nibba2018, is it ready for a review?

File: fury/ui.py [hunk 1]
```diff
@@ -4657,6 +4657,361 @@ def left_button_dragged(self, i_ren, _obj, _sub_component):
...
+            UI component.
+        size : (int, int)
+            Width and height in pixels of this UI component.
+        nb_tabs : int
+            Number of tabs to be renders.
+        active_color : tuple of 3 floats.
+            Background color of active tab panel.
+        inactive_color : tuple of 3 floats.
+            Background color of inactive tab panels.
+        draggable : {True, False}
```
Comment (skoudoro) [inline fury/ui.py:4687]: `draggable : bool` instead

**Comment by Nibba2018** [comment]
Yes @skoudoro , it is.

File: fury/ui.py [hunk 2]
```diff
@@ -4657,6 +4657,361 @@ def left_button_dragged(self, i_ren, _obj, _sub_component):
...
+            self.tabs.append(tab_panel)
+        self.update_tabs()
+
+    def _get_actors(self):
+        """ Get the actors composing this UI component.
+        """
+        actors = []
+        actors += self.parent_panel.actors
+        for tab_panel in self.tabs:
+            actors += tab_panel.actors
```
Comment (skoudoro) [inline fury/ui.py:4727]: let's make this more pythonic with:

`actors = self.parent_panel.actors + [tab_panel.actors for tab_panel in self.tabs]`

if it does not fit in 1 line, you can split it into 2 steps.

Comment (Nibba2018) [reply fury/ui.py:4727]: Hello @skoudoro , this snippet wont work as `tab_panel.actors` returns a list. Therefore after the comprehension is finished a 2D list is generated where each corresponding list contains the actors of the `tab_panel`.
Something like this:
```
[[actors of tab panel 1],
 [actors of tab panel 2],
 ...
[actors of last tab panel]]
```

Comment (skoudoro) [reply fury/ui.py:4727]: > this snippet wont work as tab_panel.actors returns a list

Oh ok, I did not see it. so no need to bother. you can keep it as it is

**Comment by Nibba2018** [comment]
Thank you @skoudoro , I have added the necessary changes. 👍

**Comment by skoudoro** [comment]
Thanks, @Nibba2018.

Does anyone want to have a look? I will wait until Thursday to merge this PR.

**Comment by skoudoro** [comment]
Thank you @Nibba2018, merging

**Pull request merged**